ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/HttpTimeoutTest.kt

```diff
@@ -415,7 +415,7 @@ class HttpTimeoutTest : ClientLoader() {
     }
 
     @Test
-    fun testConnectionRefusedException() = clientTests(listOf("Js", "native:*")) {
+    fun testConnectionRefusedException() = clientTests(listOf("Js", "native:*", "win:*")) {
```

Comment:
We need a ticket to not forget about this thing: one day we'll enable it back, right?

Reply:
Nope, the original test was muted cause this kind of timeout is not supported on windows